modules/installation-creating-aws-vpc-subnets-lz.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-local-zone.adoc (Installing a cluster on AWS with compute nodes on AWS Local Zones) 
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-creating-aws-vpc-subnets-lz_{context}"]
+= Creating subnets in Local Zones
+
+Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create the subnets in {zone-type}. Complete the following procedure for each Local Zone that you want to deploy compute nodes to.
+
+You can use the provided CloudFormation template and create a CloudFormation stack. You can then use this stack to custom provision a subnet.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS infrastructure, you must review the provided information and manually create the infrastructure. If your cluster does not initialize correctly, you might have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You opted in to the {zone-type} group.
+
+.Procedure
+
+. Go to the section of the documentation named "CloudFormation template for the VPC subnet", and copy the syntax from the template. Save the copied template syntax as a YAML file on your local system. This template describes the VPC that your cluster requires.
+
+. Run the following command to deploy the CloudFormation template, which creates a stack of AWS resources that represent the VPC:
++
+[source,terminal]
+----
+aws cloudformation create-stack --stack-name <stack_name> \ <1>
+  --region ${CLUSTER_REGION} \
+  --template-body file://<template>.yaml \ <2>
+  --parameters \
+    ParameterKey=VpcId,ParameterValue="${VPC_ID}" \ <3>
+    ParameterKey=ClusterName,ParameterValue="${CLUSTER_NAME}" \ <4>
+    ParameterKey=ZoneName,ParameterValue="${ZONE_NAME}" \ <5>
+    ParameterKey=PublicRouteTableId,ParameterValue="${ROUTE_TABLE_PUB}" \ <6>
+    ParameterKey=PublicSubnetCidr,ParameterValue="${SUBNET_CIDR_PUB}" \ <7>
+    ParameterKey=PrivateRouteTableId,ParameterValue="${ROUTE_TABLE_PVT}" \ <8>
+    ParameterKey=PrivateSubnetCidr,ParameterValue="${SUBNET_CIDR_PVT}" <9>
```

Comment:
```suggestion
$ aws cloudformation create-stack --stack-name <stack_name> \// <1>
  --region ${CLUSTER_REGION} \
  --template-body file://<template>.yaml \// <2>
  --parameters \
    ParameterKey=VpcId,ParameterValue="${VPC_ID}" \// <3>
    ParameterKey=ClusterName,ParameterValue="${CLUSTER_NAME}" \// <4>
    ParameterKey=ZoneName,ParameterValue="${ZONE_NAME}" \// <5>
    ParameterKey=PublicRouteTableId,ParameterValue="${ROUTE_TABLE_PUB}" \// <6>
    ParameterKey=PublicSubnetCidr,ParameterValue="${SUBNET_CIDR_PUB}" \// <7>
    ParameterKey=PrivateRouteTableId,ParameterValue="${ROUTE_TABLE_PVT}" \// <8>
    ParameterKey=PrivateSubnetCidr,ParameterValue="${SUBNET_CIDR_PVT}" <9>
```
Added `$` and added forward slashes to fix syntax highlighting. Preview should look like this:
<img width="732" alt="Screenshot 2024-02-15 at 12 51 45 PM" src="https://github.com/openshift/openshift-docs/assets/93731574/181622b5-7031-4058-a84d-932a8e1beab4">
There are ~3 other code blocks for the `aws cloudformation create-stack` command in this PR that need the same fix.

Reply:
Thanks, Ben. Updated